Gemfile.lock

```diff
@@ -1,13 +1,13 @@
 GIT
   remote: https://github.com/codeforamerica/honeycrisp-gem
-  revision: 3f63cc61fef1ac15947b6019eb7abe61f6a5ec8a
+  revision: 320bff4e9359bf2ba9db7236c85499589b632315
```

Comment:
Bumped the version of honeycrisp in the app now. there was a breaking change in this version that changes the default font size in honeycrisp-compact from 19 to 18, so you'll see some of those updates made in the code.

Reply:
How confident are we in this change not making the app ugly in mysterious ways? I haven't gone and validated it - would it be smart to do screenshot diffing?

Reply:
I see - it's just Honeycrisp Compact that had the breaking changes. If it doesn't affect clients, I am quite OK with breaking changes.

Reply:
@alaroia-cfa This actually solves a mystery that Anu and I were looking into last week where like ... Honeycrisp compact was significantly more compact in the docs than on our app, and it turns out its because of these changes! So we do want them and I did spot check - albeit briefly and incompletely - that there are no breaking changes.